imports/server/setup.ts

```diff
@@ -105,6 +107,32 @@ Meteor.methods({
     Settings.remove({ name: 'gdrive.credential' });
   },
 
+  setupGdriveRoot(root: unknown) {
+    check(this.userId, String);
+    check(root, Match.Maybe(String));
+    // Only let the same people that can credential gdrive configure root folder,
+    // which today is just admins
+    if (!userMayConfigureGdrive(this.userId)) {
+      throw new Meteor.Error(401, 'Must be admin to configure gdrive');
+    }
+
+    if (root) {
+      Settings.upsert(
+        { name: 'gdrive.root' },
+        { $set: { value: { id: root } } }
+      );
+
+      Meteor.defer(() => {
```

Comment:
I think that maybe this should maybe become a separate Meteor method, reachable via an optional "also migrate any of my past hunt data" that we could show in the same configuration section?  It's definitely non-obvious from the UI that filling out this setting will cause immediate side-effects on the Drive side.